robot/packages/package_manager.go

```diff
@@ -20,8 +18,6 @@ type (
 	PackageVersion string
 )
 
-const maxPackageSize = int64(50 * units.GB)
```

Comment:
🤷🏻 probably fine. I guess we'll just attempt to download modules > 50gb where we would've errored without even trying before. cc @cheukt for visibility.

Reply:
I think this also exposes us to compression bombs in tar unpacking (previously we were using this as a limit on both of compressed + decompressed size)